project.clj

```diff
@@ -104,7 +104,7 @@
   :plugins [[lein-environ "1.1.0"]                                    ; easy access to environment variables
             [lein-ring "0.11.0"                                       ; start the HTTP server with 'lein ring server'
              :exclusions [org.clojure/clojure]]                       ; TODO - should this be a dev dependency ?
-            [puppetlabs/i18n "0.8.0"]]                                ; i18n helpers
+            [puppetlabs/i18n "0.8.0"]]                                                       ; i18n helpers
```

Comment:
accidental?

Reply:
Yeah, I'll fix that up, I added the pprint plugin to test out the profile stuff, must have bumped the comment.